modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -118,8 +118,102 @@ definition {
 		}
 	}
 
-	@description = "LPS-178192. Verifies Expanded Text effects toggle can be enabled."
-	@priority = 4
+	@description = "LPS-178192."
+	@priority = 5
+	test CanBeDisabledByInstanceSettings {
+		property portal.acceptance = "true";
+
+		task ("Given user with admin privileges") {
+			Navigator.openURL();
+		}
+
+		task ("And “Enable Accessibility Menu” is enabled") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			Click(
+				key_itemName = "Accessibility",
+				locator1 = "ListGroupItem#ITEM_TEXT");
+
+			AssertChecked.assertCheckedNotVisible(
+				checkboxName = "Enable Accessibility Menu",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+		}
+
```

Comment:
Please remove this task since it's redundant. It's already done in setup Given step.